src/components/Entities/LabelList.js

```diff
@@ -19,6 +43,8 @@ export const LabelList = observer(({ regionStore }) => {
     };
   });
 
+  const expandedKeys = treeData.filter( item => !collapsedPos.includes( item.pos ) ).map( item => item.key ) ?? [];
```

Comment:
probably should put any of the new additional functionality behind the FF.

the `Tree` props `expandedKeys` and `onExpand` will have the new behaviour by default right now.